src/MySqlConnector/MySqlCommand.cs

```diff
@@ -390,8 +397,7 @@ public Task DisposeAsync()
 		if (!cancellationToken.CanBeCanceled)
 			return null;
 
-		m_cancelAction ??= Cancel;
-		return cancellationToken.Register(m_cancelAction);
+		return cancellationToken.Register(Cancel);
```

Comment:
When this code was written, not caching the delegate would result in a lot more allocations; has that changed?

Reply:
ok, i see the point, wrong correction then